stock_picking_show_lot/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds a button to the "stock.picking" form if you have the option
+"Auto Create Lots" activated in the operation type that allows you to display the
```

Comment:
Why is it in this case only ? Maybe this should be moved to CONTEXT fragment. IMHO, this module should not depend on stock_picking_auto_create_lot module.

Reply:
It doesn't really depend on "stock_picking_auto_create_lot" and the description was not correct, now it is. The button should appear when the option to create lots is active in the transaction type. Thanks